components/variations/proto/study.proto

```diff
@@ -183,6 +183,7 @@ message Study {
   // Possible Chrome operating system platforms.
   // These names must match those in tools/variations/fieldtrial_to_struct.py.
   enum Platform {
+    PLATFORM_WINDOWS         = -1;
```

Comment:
I haven't had time for a full pass but - this looks like a mistake ? s/WINDOWS/STARBOARD/ ?

Reply:
Whoops missed a file in my push. PTAL.